src/Tribe/Google/Maps_API_Key.php

```diff
@@ -47,9 +47,14 @@ public function filter_tribe_addons_tab_fields( array $addon_fields ) {
 
 			'gmaps-js-api-info-box' => array(
 				'type' => 'html',
-				'html' => '<p>' . sprintf( __( 'We highly recommend that you specify a valid %s for The Events Calendar to use. Doing this will help prevent problems with maps, especially for sites that receive a lot of traffic.',
-						'the-events-calendar' ), '<a href="https://developers.google.com/maps/documentation/javascript/get-api-key" target="_blank">' . __( 'Google Maps API key',
-							'the-events-calendar' ) . '</a>' ) . '</p>',
+				'html' => '<p>' . sprintf(
+					__(
+						'We highly recommend that you specify a valid %s for The Events Calendar to use. Doing this will help prevent problems with maps, especially for sites that receive a lot of traffic. Visit %s for a detailed explanation in how to do it.',
```

Comment:
Numbered placeholders like `%1$s` are really helpful to give translators max flexibility

Reply:
Great idea 👍